lib/textris/base.rb

```diff
@@ -53,9 +42,15 @@ def call_action
     end
 
     def render_content
-      set_instance_variables_for_rendering
-
-      render(:template => template_name, :formats => ['text'], :locale => @locale)
+      renderer = ActionController::Base.renderer.new
+
+      renderer.render(
+        template: template_name,
+        layout: false,
+        formats: [:html],
```

Comment:
```suggestion
        formats: [:text],
```

`:formats` is `[:text]` ?